### ISSUE TITLE
FEAT : National Identification Number

### ISSUE THREAD

**Issue by seon22break** [chancejs/chancejs#570]
For me it is very useful to have the possibility to generate national identification codes. In Spain the NIF is used with a specific format but in other places it is different. I think it would be good to add some of the most used identifiers.

Thanks for your time!

**Comment by leesei** [comment]
Any reference as to the format of this Identification Number?

**Comment by seon22break** [comment]
> Any reference as to the format of this Identification Number?

Yes , for example the spanish number are 8 numbers and one lettler. (this letter is calculated )

**Comment by leesei** [comment]
How is the letter calculated?
By reference I mean clear specification and algorithm so we can implement it.
Your reply isn't helping.
